src/models/StandaloneSimilarity.py

```diff
@@ -0,0 +1,21 @@
+import random
+
+from cleaning_functions import pandas_vector_to_list
+from job_description_feature_extraction import get_tfidf_similarity, extract_relevant_documents
+from models.BaseModel import BaseModel
+
+
+class StandaloneSimilarity(BaseModel):
```

Comment:
this could be an example implementation of an actual prediction model

for now we just return some random data to conform with the interface